common/json.go

```diff
@@ -29,6 +29,24 @@ type JSONHTTPClient struct {
 	ErrorHandler ErrorHandler            // optional error handler. If not set, then the default error handler is used.
 }
 
+type JSONHTTPResponse struct {
+	bodyBytes []byte
```

Comment:
Can you add a comment that Body is the same data as bodyBytes, but unmarshalled into JSON?